configuration/analyzer_config.json

```diff
@@ -229,6 +229,15 @@
     "python_module": "file_info.FileInfo",
     "soft_time_limit": 30
   },
+  "Floss": {
+    "type": "file",
+    "description": "Static Analysis of Binaries",
```

Comment:
I would say something more useful for the user. Ex: "advanced strings extraction from FireEye"

Reply:
Sorry, I wasn't aware of this type of analyzers, So, I directly kept a file And Sure. I'll make the changes.